torch/_prims_common/__init__.py

```diff
@@ -1801,10 +1800,5 @@ def set_torch_state_tensor(seed, offset):
         torch.cuda.set_rng_state(new_state)
 
     @staticmethod
-    def advance_torch_state(relative_offset):
-        rng_state = torch.cuda.get_rng_state()
-        # Rng state is [64-bit seed, 64-bit offset]
-        seed = rng_state[0:8].view(dtype=torch.int64)[0]
-        offset = rng_state[8:].view(dtype=torch.int64)[0]
-        new_offset = offset + relative_offset
-        CUDARngStateHelper.set_torch_state_tensor(seed, new_offset)
+    def set_new_offset(relative_offset):
+        torch.cuda._set_rng_state_offset(relative_offset.item())
```

Comment:
Does this get traced?

Reply:
No, this is post-fix. After the fwd or bwd graph is executed, this is called separately outside to advance the state.